Thanks.  Reviewed b0bb65f.

* [x] the new error message has extra whitespace; please fix; here's what it looks like in log;
```log
1562732722.234531 ERROR root: Please install the libraries python-scipy,     python-numpy and python-matplotlib through apt or     your favourite package manager, to display the line graph
```

* [x] new variable `import_plot_error` does not explain what it is for; please choose a better name?  Imagine yourself as a new developer reading one of the _references_ to the new variable, such as line 93, can you figure out what it is for without having to read where it is defined?

* [x] flake8 messages occur in areas where you have changed the code;

```log
./activity.py:449:13: E128 continuation line under-indented for visual indent
./activity.py:470:80: E501 line too long (83 > 79 characters)
./activity.py:471:80: E501 line too long (83 > 79 characters)
./activity.py:472:80: E501 line too long (83 > 79 characters)
./activity.py:495:80: E501 line too long (81 > 79 characters)
./activity.py:502:28: E128 continuation line under-indented for visual indent
./activity.py:503:28: E128 continuation line under-indented for visual indent
./activity.py:505:28: E128 continuation line under-indented for visual indent
./activity.py:506:28: E128 continuation line under-indented for visual indent
./activity.py:508:28: E128 continuation line under-indented for visual indent
./activity.py:509:28: E128 continuation line under-indented for visual indent
```

* [x] the truncation of the axis titles continues to occur; this is on Ubuntu 18.04 with display dimensions of 1024x768; please reproduce those dimensions and test?  I've looked at your changes so far, and it seems unlikely to be something you've done, but rather a latent flaw in the source code before you worked on it, so what you did was expose the flaw.  Although I'm not entirely sure why you declare figsize=(100,100) nor why you add both children of the Gtk.Box with same flags.

--- a/activity.py
+++ b/activity.py
@@ -446,7 +446,8 @@ class LineGraph(FigureCanvas):
         self.axes.set_xticks(self._x_axis, minor=True)
 
         self.disp_args = {'size': 'x-large',
-            'family': 'monospace', 'style': 'italic'}
+                          'family': 'monospace',
+                          'style': 'italic'}
 
         self.calc()
         self.calculate_graph_values()
@@ -467,9 +468,12 @@ class LineGraph(FigureCanvas):
             each_day = today + timedelta(days=diff - 8)
             self.labels.append(str(each_day))
             dif = each_day - birth
-            self.p.append(int(sin(2 * 3.14159 * dif.days / 23) * self._scale * -1))
-            self.e.append(int(sin(2 * 3.14159 * dif.days / 28) * self._scale * -1))
-            self.i.append(int(sin(2 * 3.14159 * dif.days / 33) * self._scale * -1))
+            self.p.append(int(sin(2 * 3.14159
+                          * dif.days / 23) * self._scale * -1))
+            self.e.append(int(sin(2 * 3.14159
+                          * dif.days / 28) * self._scale * -1))
+            self.i.append(int(sin(2 * 3.14159
+                          * dif.days / 33) * self._scale * -1))
 
     def calculate_graph_values(self):
         self.axes.clear()
@@ -492,21 +496,22 @@ class LineGraph(FigureCanvas):
 
         self.axes.grid(True)
         x_major = [''] + self.labels[1::2]
-        x_minor_labels = self.axes.set_xticklabels(self.labels[0::2], minor=True)
+        x_minor_labels = \
+            self.axes.set_xticklabels(self.labels[0::2], minor=True)
         x_major_labels = self.axes.set_xticklabels(x_major, minor=False)
 
         for label in x_major_labels:
             match = [None, None, None]
             try:
                 match[2] = int((label.get_text()
-                           .encode('ascii', 'ignore')
-                           .split('-')[0]))
+                                     .encode('ascii', 'ignore')
+                                     .split('-')[0]))
                 match[1] = int((label.get_text()
-                           .encode('ascii', 'ignore')
-                           .split('-')[1]))
+                                     .encode('ascii', 'ignore')
+                                     .split('-')[1]))
                 match[0] = int((label.get_text()
-                           .encode('ascii', 'ignore')
-                           .split('-')[2]))
+                                     .encode('ascii', 'ignore')
+                                     .split('-')[2]))
             except ValueError:
                 continue
             if match[0] == t[0] and match[1] == t[1] and match[2] == t[2]:

